apps/spikeConverter.cpp

```diff
@@ -93,6 +93,13 @@ int main( int argc, char* argv[] )
                   << brion::Version::getString() << std::endl;
         return EXIT_SUCCESS;
     }
+    if( vm["input"].as< std::string >() == vm["output"].as< std::string >( ))
+    {
+        std::cerr << "Cowardly refusing to convert "
+                  << vm["input"].as< std::string >() << " onto itself"
+                  << std::endl;
+        return EXIT_FAILURE;
+    }
```

Comment:
You're right about the output parameter, I didn't consider that, but I disagree for the input, I couldn't care less about the test data when using this tool.
Regarding being able to simply do "spikeConverter in out" it's not laziness at all, positional arguments for necessary arguments leaving an optional one with a default at the end it's a pretty common CLI pattern. Many tools work like that, to name a few: cp, mkdir, cat, convert, grep, even for gcc you can do "gcc file.cpp".

But the most important detail is that removing positional arguments was a breaking change.